include/base/OpenMCCellAverageProblem.h

```diff
@@ -839,7 +845,7 @@ class OpenMCCellAverageProblem : public OpenMCProblemBase
    * Whether to map density according to each individual OpenMC cell (in which case an
    * error is thrown if you don't have a unique material in each cell) or by material.
    */
-  const bool & _map_density_by_cell;
+  bool _map_density_by_cell;
```

Comment:
Now that we're making our own copy of this variable, it's going to be out of sync with the one we get using the MOOSE input parsing system. I don't generally think this will be a problem based on the changes below, but is there a way to programmatically set the value of that variable inside the MOOSE parameters system instead?

Reply:
And if we don't feel like it's worth the trouble, this comment thread can be a placeholder/reminder in case it does become a problem in the future.

Reply:
> I don't believe that we can actually change the value in the MOOSE parameters system

This is the piece of info I was missing. Didn't know we couldn't change those! Makes sense to take ownership of the value here in that case, though it is surprising we can't change those values through some call into the MOOSE params.

Reply:
Maybe I don't fully understand what you're proposing? When fetching from the input file, parameters are grabbed as `const &`, so there's no way to change the values if they're const.